scio-test/src/test/scala/com/spotify/scio/transforms/AsyncLookupDoFnTest.scala

```diff
@@ -69,6 +77,22 @@ class AsyncLookupDoFnTest extends PipelineSpec {
     ()
   }
 
+  "BaseAsyncDoFn" should "deduplicate simultaneous lookups on the same item" in {
+    val n = 100
+    val sc = ScioContext(PipelineOptionsFactory.fromArgs("--targetParallelism=1").create())
```

Comment:
is there a specific reason the parallelism is being limited to 1?

Reply:
Just to increase contention in the test.